Implemented upload, timeline, region selection function.

harry2636 opened the pull request:
- Upload music files to web.

- Add timeline for uploaded music file.

- Add Region selection by dragging.

* Moved wavesurfer.js plugin codes to repository to customize for our project.

harry2636 committed:
Add upload file functionality.

harry2636 committed:
Add timeline to waveform.

harry2636 committed:
Add region selecting funcion.
Moved wavesurfer plugins to project repository to customize the code for
our project.

jaemin-shin commented:
Overall, looks good to me!
I have few questions:
1. How do I un-select the selected region?
2. When the '드래그로 영역지정' button is 'clicked' state, and if I drag on a selected region, the region moves. Is this intended?

I think I still need to see more details of the code..

harry2636 committed:
Changed region selection behavior.
Region selection is always enabled.
Can select only one region.
Delete selected region if click occurs.

harry2636 commented:
I changed the behavior of the region selecting function.
Please check it.

- Region selection is always enabled.
- Can select only one region.
- Delete selected region if click occurs.
- Disabled region move.

harry2636 committed:
Disabled region move.
